src/omv/imlib/framebuffer.h

```diff
@@ -34,6 +34,7 @@ typedef struct framebuffer {
     volatile int32_t tail;
     bool check_head;
     int32_t sampled_head;
+    uint32_t frame_size;
```

Comment:
Might want to name this something else since sensor.framesize means something completely different.

Reply:
`raw_buffer_size` is also very misleading. It seems to be the vbuffer buffer size + header but it's called raw.

Reply:
`frame_size` doesn't need to be stored, it can be passed as an arg, but I want to keep the function accepting a single arg (count).

Reply:
Yeah, I was just suggesting giving it a better name.

Reply:
I can't think of better names for this, it is a frame size. However, I'm still thinking about just passing it as a second arg to `set_buffers`. The reason for this is that if you ever call this function with anything other than `1` you must remember to set `frame_size` first, otherwise it will use whatever was there.